addon/styles/_frost-button.scss

```diff
@@ -129,10 +131,14 @@ $frost-color-button-tertiary-active-bg: #e7eef3 !global;
     }
   }
 
+  // ----------------------
   // Sizes
+  // ----------------------
+
+  // size = 'extra-large'
```

Comment:
At the moment it is not being used.  He wants to keep it though. I kept the code, but did not expose this in the readme and the demo.